rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+- Standard/specification for this feature.
+- A list of tests on wpt.fyi which will be used to score progress towards interoperability on the feature.
+
+#### Investigation efforts
+
+If an area doesn't have a good spec and test suite it's not possible to use wpt.fyi test results to judge progress. Instead, some other work is needed to bring the area to state where test results can be used.
+
+The following information should be provided in a proposal:
+
+- Is this area a problem for web developers? For example, survey data and browser bug counts can help.
+- Is this area a problem for users?
+- How widely used is this feature? For example, [use counters](https://www.chromestatus.com/metrics/feature/popularity), [HTTP Archive](https://httparchive.org/) and [State of CSS](https://2021.stateofcss.com/en-US/features/) can help.
+- Standard/specification for this feature, if any.
+- A list of tasks which should be completed as part of the effort. This could be producing a report for a working group, a list of spec bugs which should be resolved, or tests/infrastructure which need to be created. A completed investigation effort should unblock the feature for inclusion as a regular focus area in the future.
+
+### Proposal selection
+
+Proposal selection (November 1 to November 30) will be done using a positions process similar to [Interop 2022](https://github.com/web-platform-tests/interop-2022/issues/38) but with some changes. The process is:
+
+- Each participating organization takes one of 3 positions on each proposal: support, neutral, or oppose.
+- A proposal is accepted if there is consensus, meaning support from at least two participating organizations and no opposition.
+- The team reviews the accepted proposals and can decide to group multiple proposals, or to drop proposals. Both actions require team consensus.
+- The final list of accepted proposals is decided by team consensus no later than November 30.
+
+The purpose of the second round of grouping and optionally dropping proposals is to avoid sub-proposals as in the Interop 2022 process. Instead, proposals will be as specific as reasonably possible, and can be grouped into themes afterwards. The expectation is that proposals will only be dropped if there is consensus that they don't make sense in light of which other proposals were accepted.
+
+### Updating the tests
+
+No test suite is perfect, and there will likely be a need to add, remove or rewrite tests during the course of the year. To suggest changes, file an issue on the [Interop repo](https://github.com/web-platform-tests/interop). The team will make decisions based on consensus, see [governance](#governance).
+
+### Metrics
+
+Per-area and overall metrics will be defined in a similar fashion to [Interop 2022](./interop_2022.md#metrics). The number of areas and their contribution to the total score will be decided in the proposal selection phase.
```

Comment:
I think in hindsight some of this hasn't actually worked out too well in Interop 2022; for example with the Web Compat focus area: https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2022-webcompat

This only has seven tests in total, four in /css/css-text-decor (for https://github.com/web-platform-tests/interop-2022/issues/23), one in /css/CSS2/floats (for https://github.com/web-platform-tests/interop-2022/issues/33), one in /dom (for https://github.com/web-platform-tests/interop-2022/issues/26), and one in /url (for https://github.com/web-platform-tests/interop-2022/issues/18).

Given this distribution, we've ended up with one of the interop bugs accounting for 4/7ths of the score.

Reply:
Do you think the RFC should say something more about the scoring than that we'll decide later?

Reply:
Would it be enough to write "similar (but not necessarily identical)"?

I agree that with something where the focus area is defined from a number of sub-proposals it doesn't necessarily make sense to weigh each sub-proposal equally (I think for webcompat we planned to have some additional tests, which has made the problem worse in that case).

Reply:
I'll add in "but not necessarily identical" 👍

Reply:
```suggestion
Per-area and overall metrics will be defined in a similar (but not necessarily identical) fashion to [Interop 2022](./interop_2022.md#metrics). The number of areas and their contribution to the total score will be decided in the proposal selection phase.
```